modules/catalog/src/main/java/org/apache/ignite/internal/catalog/commands/CatalogUtils.java

```diff
@@ -593,28 +593,28 @@ public static HybridTimestamp clusterWideEnsuredActivationTsSafeForRoReads(
         return defaultZone != null ? defaultZone.id() : null;
     }
 
-    /**
-     * Return {@code true} if a function with given name is allowed to be used as functional default for a column, {@code false} otherwise.
-     */
-    static boolean isSupportedFunctionalDefault(String functionName) {
-        return FUNCTIONAL_DEFAULT_FUNCTIONS.contains(functionName.toUpperCase());
-    }
-
     /**
      * Check if provided default value is a constant or a functional default of supported function, or fail otherwise.
      */
-    static void ensureSupportedDefault(String columnName, @Nullable DefaultValue defaultValue) {
+    static void ensureSupportedDefault(String columnName, @Nullable DefaultValue defaultValue, ColumnType columnType) {
```

Comment:
```suggestion
    static void ensureSupportedDefault(String columnName, ColumnType columnType, @Nullable DefaultValue defaultValue) {
```
to simplify future merge

Reply:
Fixed.